src/vm/wren_vm.c

```diff
@@ -5,6 +5,7 @@
 #include "wren_common.h"
 #include "wren_compiler.h"
 #include "wren_core.h"
+#include "wren_primitive.h"
```

Comment:
Doesn't seem like the right position for this include, assuming the alphabetical order was intensional.

Reply:
Oh yes, so it seems. Fixed :)